java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -92,8 +95,19 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
 
         baseTransaction.configResetStat();
 
-        // TODO
-        // baseTransaction.pfadd(hllKey1, new String[0]);
+        baseTransaction
+            .pfadd(hllKey1, new String[0])
+            .pfadd(hllKey1, new String[] {"a", "b", "c"})
+            .pfadd(hllKey2, new String[] {"b", "c", "d"})
+            .pfadd(hllKey1, new String[0])
+            .pfadd(hllKey3, new String[0]);
+
+        baseTransaction
+            .pfcount(new String[] { "{key}:" + UUID.randomUUID()}) // TODO rename after #160
```

Comment:
We should have just one test for transactions we can test functionality in shared commands. (This is consistent with Py and Node and Java)